MvvmCross.Forms/Platform/Mac/Platform/MvxFormsMacSetup.cs

```diff
@@ -63,7 +63,7 @@ public MvxFormsApplication FormsApplication
 
         protected abstract MvxFormsApplication CreateFormsApplication();
 
-        protected override IMvxMacViewPresenter CreatePresenter()
+        protected override IMvxMacViewPresenter CreateViewPresenter()
```

Comment:
Do we need this breaking change? `CreateViewPresenter` is much cleaner, but we don't want users to hate us / not to move forward to 6.0 because of tons of breaking changes. Maybe keep both and mark CreatePresenter as obsolete?

Reply:
I think we should just break it. It's 6.0 and moving to netstandard will changes a lot of things anyways.

Reply:
I may be nice to track all these breaking changes in a doc somewhere so it can be linked to in the release notes for 6.0 to add in the migration?

Reply:
Yes, we definitely need to document them. We will create a document when we release 6.0 containing a migration guide.